packages/web/components/MegaMenu/MegaMenu.tsx

```diff
@@ -52,13 +55,28 @@ import XPEarned from 'assets/menuIcon/xpearned.svg';
 import Youtube from 'assets/menuIcon/youtube.svg';
 import { MetaLink } from 'components/Link';
 import { PlayerAvatar } from 'components/Player/PlayerAvatar';
-import { Player } from 'graphql/autogen/types';
+import {
+  GuildFragmentFragment,
+  Player,
+  PlayerFragmentFragment,
+} from 'graphql/autogen/types';
 import { useUser, useWeb3 } from 'lib/hooks';
 import { useRouter } from 'next/router';
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
+import { distinctUntilChanged, Subject } from 'rxjs';
+// import { forkJoin, from } from 'rxjs';
+import { debounceTime, filter } from 'rxjs/operators';
+// import {  switchMap } from 'rxjs/operators';
 import { MenuLinkItem, MenuLinkSet, MenuSectionLinks } from 'utils/menuLinks';
-import { getPlayerURL } from 'utils/playerHelpers';
+import {
+  getPlayerImage,
+  getPlayerName,
+  getPlayerURL,
+} from 'utils/playerHelpers';
 
+import SearchIcon from '../../assets/search-icon.svg';
+import { getPlayersByText } from '../../graphql/getPlayers';
+import { getGuildsByText } from '../../graphql/queries/guild';
```

Comment:
```suggestion
import SearchIcon from 'assets/search-icon.svg';
import { getPlayersByText } from 'graphql/getPlayers';
import { getGuildsByText } from 'graphql/queries/guild';
```

Reply:
what configuration in the repo is required to replicate this(not writing relative path) ?

Reply:
I'm pretty sure it's [`baseUrl` in `tsconfig.json`](//github.com/MetaFam/TheGame/blob/develop/packages/web/tsconfig.json#L10).